ts/src/main/scala/ch/epfl/ts/traders/Trader.scala

```diff
@@ -61,10 +60,6 @@ abstract class Trader(val uid: Long, marketIds : List[Long],val parameters: Stra
       println("Got a GetTraderParameters")
       sender ! TraderIdentity(self.path.name, uid, companion, parameters)
     }
-    
-    case TheTimeIs(t) => {
```

Comment:
I do believe we need all traders to handle `TheTimeIs`  messages, why was it removed?